### PR TITLE
Dia examples new methods

### PR DESCRIPTION
**Pull request by vinnydavies**

### CODE REVIEW

File: vimms/Controller/dia.py [hunk 1]
```diff
@@ -176,3 +171,96 @@ def _process_scan(self, scan):
             self.scan_number += len(precursor_mz_list) + 1
             self.next_processed_scan_id = self.scan_number
         return new_tasks
+
+
+class DiaController(Controller):
```
Comment (sdrogers) [inline vimms/Controller/dia.py:120]: Could you add some comments to our future self to just clarify the difference between this and the other DIA ones?

**Review by sdrogers — CHANGES_REQUESTED** [review]

File: vimms/Controller/dia.py [hunk 2]
```diff
@@ -119,12 +120,6 @@ def __init__(self, min_mz, max_mz,
         self.scan_number = self.initial_scan_id
         self.exp_info = []  # experimental information - isolation windows
 
-    def handle_acquisition_open(self):
```
[the remark this answers is not in the document]
Comment (vinnydavies) [reply vimms/Controller/dia.py]: they got moved to environment when joe was sorting out the contact closure stuff, so no longer needed here

File: vimms/Controller/dia.py (hunk 1 above)
Comment (vinnydavies) [reply vimms/Controller/dia.py:120]: comments have been added in the latest commit

File: vimms/Controller/dia.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (sdrogers) [reply vimms/Controller/dia.py]: cool

**Pull request merged**